modules/utxo-coredao/src/transaction.ts

```diff
@@ -0,0 +1,121 @@
+// Source: https://docs.coredao.org/docs/Learn/products/btc-staking/design
+export const CORE_DAO_TESTNET_CHAIN_ID = Buffer.alloc(2, 0x1115);
+export const CORE_DAO_MAINNET_CHAIN_ID = Buffer.alloc(2, 0x1116);
+export const CORE_DAO_SATOSHI_PLUS_IDENTIFIER = Buffer.alloc(4, 0x5341542b);
+
+/** As of v2, this is the construction of the OP_RETURN:
```

Comment:
move this as a `/* */` comment inside the function body - the `/**` comment should focus on the interface, not the impl